### DISCUSSION TITLE
WOCHE 02 (Sem4) - RMMM-Tabelle und Updates

### DISCUSSION THREAD

**Discussion by DossohMael** [mausio/corgi-shop-doc#12] [Blogs]
Dieser Blog Post beinhaltet unsere [RMMM-Tabelle](https://github.com/mausio/corgi-shop-doc/blob/main/rmmm/RMMM_Tabelle.md) sowie ein kurzes Update über die neusten Anpassungen der App.

# Die RMMM-Tabelle
Das RMMM ist ein Werkzeug, das verwendet wird, um potenzielle Risiken bei einem Projekt zu identifizieren, zu bewerten und zu verwalten. Diese Spalten der Tabelle enthalten Informationen über die identifizierten Risiken wie ihre Art, Eintrittswahrscheinlichkeit, Auswirkungen, Maßnahmen zur Risikominimierung und Verantwortlichkeiten festgehalten werden. 
Sie dient als zentrales Dokument, um das Risikomanagement im Projekt zu steuern und zu dokumentieren.

# Neuste Anpassungen der App
Das Backend wurde einem Server hinzugefügt um statische URL-Adressen für das Frontend zu gewährleisten.

**Comment by Fefke** [comment]
Saubere und kompakte RMMM-Tabelle 💪 
Ich frag mich nur wofür ihr den Risk-Score 10 vorbehaltet,
ist ein Datenschutz-Vorfall - besonders mit Zahlungsinformationen - nicht der Supergau? ^^

Ansonsten Süper, weiter so!

**Reply by DossohMael** [reply]
Vielen Dank für die Rückmeldung!
Gerade wollten wir die höchst mögliche Risikostufe noch nicht für Datenschutz-Probleme vergeben, 
da wir selbst nur sehr wenige Daten der Nutzer speichern und das externe Zahlungssystem noch nicht ausgewählt haben.
